lib/MusicBrainz/Server/Report/ArtistsThatMayBePersons.pm

```diff
@@ -21,23 +21,23 @@ WITH groups_entity0 AS (
          JOIN link ON link.id = laa.link
          JOIN link_type ON link_type.id = link.link_type
          WHERE artist.type IS DISTINCT FROM 1
-         AND link_type.name IN ('member of band', 'collaboration', 'conductor position', 'subgroup')),
+         AND link_type.name IN ('member of band', 'collaboration', 'conductor position', 'artistic director', 'composer-in-residence', 'subgroup')),
```

Comment:
We should probably use link type IDs here rather than names. :) (Could be a future improvement though.)